data-mermaid/mermaid-webapp
# Add methods filter on submitted table

**summary**

Previously this was implemented on the collecting table. I decided to add separate prs for each table to break it up + each one requires something slightly different

**to test**

1. go to a project with submitted transects
2. go to submitted table
3. select items to fliter
4. refresh
5. ensure persist works
